lib/cmd/fh3/projects.js

```diff
@@ -52,6 +52,10 @@ function projects(argv, cb) {
 function listProjects(args, cb) {
   common.listProjects(function (err, projs) {
     if (err) return cb(err);
+    //Allow filtering by author email
+    if(args[1]){
```

Comment:
if you change author-email to just email you would be able to refer to them using args.email